File: atv_pratica1/src/LocaLisa.java
import java.util.ArrayList;

public class LocaLisa {
    Repositorio repo;
    public LocaLisa(Repositorio repo) {
        this.repo = repo;
    }

    public void addVeiculo(Veiculo veiculo) {
        this.repo.addVeiculo(veiculo);
    }

    public void imprimirVeiculos() {
        this.repo.imprimirVeiculos();
        this.repo.imprimirMarcasMaisRelevantes();
    }

    public void imprimirCarrosDisponiveis() {
        this.repo.imprimirCarrosDisponiveis();
    }

    public void imprimirCarrosAlugados() {
        this.repo.imprimirCarrosAlugados();
    }

}

File: atv_pratica1_kotlin/src/main/kotlin/LocaLisa.kt
class LocaLisa(var repo: Repositorio) {
    fun addVeiculo(veiculo: Veiculo?) {
        repo.addVeiculo(veiculo!!)
    }

    fun imprimirVeiculos() {
        repo.imprimirVeiculos()
        repo.imprimirMarcasMaisRelevantes()
    }

    fun imprimirCarrosDisponiveis() {
        repo.imprimirCarrosDisponiveis()
    }

    fun imprimirCarrosAlugados() {
        repo.imprimirCarrosAlugados()
    }
}
